example/README.md

```diff
@@ -0,0 +1,3 @@
+This is a simple live demo of `restul-react` components primarily used to test features.
+
+Run it by getting a global Parcel by running `npm i -g parcel`, then running `parcel example/index.html` from root.
```

Comment:
I think we can add parcel as a devDependency and it should be fine.

Reply:
Oh right, makes sense :)